website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -8040,6 +8042,8 @@ input.apply(
 
 #### 13.2.2. Using cross-language transforms in a Python pipeline
 
+For Beam versions 2.60.0+, please follow [this guide](sdks/python-custom-multi-language-pipelines-guide.md#use-the-portable-transform-in-a-python-pipeline) instead.
+
```

Comment:
Does this section actually need this disclaimer? I think consuming schema transforms is basically the same, right/nothing has changed for this section?